### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by girivelli**
Kfx labs  smart community platform -applications

### CODE REVIEW

File: applications/kfxchart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```
Comment (abjyoti) [inline applications/kfxchart/Chart.yaml:1]: Copyright and License headers are missing in many files.

**Review by abjyoti — CHANGES_REQUESTED** [review]
Copyright and License Headers are missing.

**Comment by girivelli** [comment]
Hi,
I have added the header files and licence for Kfxchart  repository, Please
review and approve
Best Regards,
--Giri.V

**Comment by subhojyoti-intel** [comment]
Kindly commit the latest changes made by you.

**Comment by girivelli** [comment]
Sure i will get it done by this weekend.
Giri

**Comment by girivelli** [comment]
Pull request to merge initiated . Please  help me to merge .
attached the screenshot

File: applications/kfxchart/helmchart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/helmchart/templates/deployment.yaml:1]: "#" is missing at the beginning of the line

File: applications/kfxchart/helmchart/templates/hpa.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/helmchart/templates/hpa.yaml:1]: "#" is missing at the beginning of the line

File: applications/kfxchart/helmchart/templates/ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/helmchart/templates/ingress.yaml:1]: "#" is missing at the beginning of the line

File: applications/kfxchart/helmchart/templates/service.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/helmchart/templates/service.yaml:1]: "#" is missing at the beginning of the line

File: applications/kfxchart/helmchart/templates/tests/test-connection.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/helmchart/templates/tests/test-connection.yaml:1]: "#" is missing at the beginning of the line

File: applications/kfxchart/helmchart/values.yaml [hunk 7]
```diff
@@ -0,0 +1,89 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/helmchart/values.yaml:1]: "#" is missing at the beginning of the line

File: applications/kfxchart/readme.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+```
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2022 Kfxlabs and circuits private Limited
```
Comment (abjyoti) [inline applications/kfxchart/readme.md:3]: "#" is missing at the beginning of the line

File: applications/kfxchart/readme.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+```
+SPDX-License-Identifier: Apache-2.0
```
Comment (abjyoti) [inline applications/kfxchart/readme.md:2]: "#" is missing at the beginning of the line

**Review by abjyoti — CHANGES_REQUESTED** [review]
Please resolve the comments.

**Review by abjyoti — APPROVED** [review]
LGTM... My comments are resolved.

**Pull request merged**